includes/database/class-edd-db-table-notes.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Notes Table: EDD_DB_Table_Notes class
+ *
+ * @package Plugins/EDD/Database/Object
+ */
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+if ( class_exists( 'WP_DB_Table' ) ) :
+/**
+ * Setup the global "edd_notes" database table
+ *
+ * @since 3.0.0
+ */
+final class EDD_DB_Table_Notes extends WP_DB_Table {
+
+	/**
+	 * @var string Table name
+	 */
+	protected $name = 'edd_notes';
+
+	/**
+	 * @var string Database version
+	 */
+	protected $version = 201801060001;
+
+	/**
+	 * @var boolean This is not a global table
+	 */
+	protected $global = false;
+
+	/**
+	 * Setup the database schema
+	 *
+	 * @since 3.0.0
+	 */
+	protected function set_schema() {
+		$this->schema = "id bigint(20) unsigned NOT NULL auto_increment,
+			object_id bigint(20) unsigned NOT NULL default '0',
+			object_type varchar(20) NOT NULL default '',
+			author_id bigint(20) unsigned NOT NULL default '0',
+			author_ip varchar(60) unsigned NOT NULL default '',
```

Comment:
Mentioned previously in the commit but not sure `author_ip` is necessary for a note as it has not been used previously and I can't seem to think why it would be necessary considering only users with certain roles will be able to make notes on certain objects (i.e. payments, discounts).